test/unit/specs/tour.spec.js

```diff
@@ -2,6 +2,7 @@ import tour from '@/tour'
 import sinon from 'sinon'
 import Vuex from 'vuex'
 import _ from 'lodash'
+import magic from '@/bus'
```

Comment:
♭ Thank you, driver, for getting me here / You'll be an inspector, have no fear ♯